apps/marketplace/reducers/briefReducers.js

```diff
@@ -20,13 +21,25 @@ const defaultBriefState = {
   isDuplicate: null,
   brief: {},
   briefResponses: [],
+  sections: [],
   specialistName: '',
   specialistNumber: 1,
+  title: '',
   addAnotherSpecialist: false
 }
 
 const briefReducer = (state = defaultBriefState, action) => {
   switch (action.type) {
+    case BRIEF_OVERVIEW_SUCCESS:
+      return {
+        ...state,
+        framework: action.framework,
+        loadBriefOverviewSuccess: true,
+        loadBriefOverviewErrored: false,
+        lot: action.lot,
+        sections: action.sections,
+        title: action.title
```

Comment:
would it be nicer to scope all these to a `overview`  attribute? so components access it in the state via `state.brief.overview.title` vs `state.brief.title`. It's probably safe not to worry but it may help give context to the state values when referenced in a connected component.